x/auth/autocli.go

```diff
@@ -80,8 +80,12 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			Service: authv1beta1.Msg_ServiceDesc.ServiceName,
 			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
 				{
-					RpcMethod: "UpdateParams",
-					Skip:      true, // skipped because authority gated
+					RpcMethod:      "UpdateParams",
+					Use:            "update-params-proposal [params]",
+					Short:          "Submit a proposal to update auth module params. Note: the entire params must be provided.",
+					Example:        fmt.Sprintf(`%s tx auth update-params-proposal '{ "max_memo_characters": 0, "tx_sig_limit": 0, "tx_size_cost_per_byte": 0, "sig_verify_cost_ed25519": 0, "sig_verify_cost_secp256k1": 0 }'`, version.AppName),
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "params"}},
+					GovProposal:    true,
```

Comment:
this is optional, if the authority is set to an multsig how would this change the flow of logic here?

Reply:
is there a flag allowing users to set this to false?

Reply:
We can add a flag and skip the gov proposal. I initially didn't do it, as it already didn't existed, but we can easily support this yes.

Reply:
Majority of user will have their module authority be gov so that's why it's done like that.